src/pages/guide/integration/attributes/data-fixture.md

```diff
@@ -16,7 +16,7 @@ To set up a date fixture, use the `DataFixture` attribute.
 
 ```php?start_inline=1
 #[
-    DataFixture(string $type, array $data = [], ?string $as = null)
+    DataFixture(string $type, array $data = [], string $count = 1, string $scope = 'default', ?string $as = null)
```

Comment:
The current order does not match the constructor signature.
```suggestion
    DataFixture(string $type, array $data = [], ?string $as = null, ?string $scope = null, int $count = 1)
```

I am glad that PHP8 introduced named arguments that allows passing values to optional arguments independently of their positions